src/style.css

```diff
@@ -0,0 +1,39 @@
+
+.body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 96px;
+  height: 96px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:nth-child(-n+1)::before,
+.stars--2 .stars__star:nth-child(-n+2)::before,
+.stars--3 .stars__star:nth-child(-n+3)::before,
+.stars--4 .stars__star:nth-child(-n+4)::before,
+.stars--5 .stars__star:nth-child(-n+5)::before {
```

Comment:
why do you need the before here, it looks redundant
```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
```

Reply:
@etojeDenys  after implementing your suggestion stars look like this
<img width="75" alt="Capture" src="https://github.com/mate-academy/layout_stars/assets/105675637/339cc92d-4536-4310-89be-e13521f6110b">